charts/nginz/templates/servicemonitor.yaml

```diff
@@ -0,0 +1,19 @@
+{{- if .Values.metrics.serviceMonitor.enabled }}
```

Comment:
```suggestion
{{- if .Values.metrics.serviceMonitor.enable }}
```

Reply:
`.Values.metrics.serviceMonitor.enabled` seems to be the more commonly used switch for this (and also the one we use in other charts), so I updated `values.yaml` to set `metrics.serviceMonitor.enable`, too.